src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -243,7 +246,8 @@ export default class DefaultAudioVideoController
     } else {
       this.meetingSessionContext.transceiverController = new DefaultTransceiverController(
         this.logger,
-        this.meetingSessionContext.browserBehavior
+        this.meetingSessionContext.browserBehavior,
+        videoConnectionOnlyMode
```

Comment:
Don't we have to pass in videoConnectionOnlyMode to SimulcastTranceiverController too?

Reply:
good catch, will fix.